templates/zerver/help/configure-authentication-methods.md

```diff
@@ -0,0 +1,35 @@
+# Configure authentication methods
+
+{!admin-only.md!}
+
+You can configure your organization settings to allow users to authenticate
+using passwords, Google/GitHub OAuth, LDAP (currently on premise only), and/or
+various other SSO methods (also currently on premise only).
+
+    !!! tip ""
```

Comment:
![screenshot at dec 29 22-35-13](https://user-images.githubusercontent.com/15116870/34451883-a1600aa8-ece8-11e7-8ff8-5fa74f7cc651.png)

Did you try viewing this file in your local/remote dev environment? This looks broken to me. (Try removing the indents)